README.md

```diff
@@ -1,19 +1,98 @@
-### Nightly Build Status (develop branch)
+# Hydroshare _(hydroshare)_
 
-| Workflow | Clean | Build/Deploy | Unit Tests | Flake8
-| :----- | :--- | :--- | :-------- | :------------ | :----------- |
-| [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-workflow/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-workflow/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-clean/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-clean/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-deploy/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-deploy/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-test/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-test/) | [![Build Status](http://ci.hydroshare.org:8080/job/nightly-build-flake8/badge/icon?style=plastic)](http://ci.hydroshare.org:8080/job/nightly-build-flake8/) |
+HydroShare is a collaborative website being developed for better access to data and models in the hydrologic sciences.
```

Comment:
I suggest:

HydroShare is a website and hydrologic information system for sharing hydrologic data and models aimed at giving hydrologists the cyberinfrastructure needed to innovate and collaborate in research to solve water problems.